src/nonce.ts

```diff
@@ -0,0 +1,48 @@
+import { ApiPromise } from '@polkadot/api'
+import { TypeRegistry } from '@polkadot/types'
+import { hexToU8a } from '@polkadot/util'
+
+import { Config } from './schema'
+import { runTask, taskHandler } from './executor'
+import { setup } from './setup'
+
+export const nonce = async (argv: Config) => {
+  const accountId = argv['accountId'] as `0x${string}`
+
+  const context = await setup(argv)
+  const wasm = await context.chain.head.wasm
+  const block = context.chain.head
+  const parent = await block.parentBlock
+
+  const getNonce = async () => {
+    const result = await runTask(
+      {
+        wasm,
+        calls: [['AccountNonceApi_account_nonce', accountId]],
+        storage: [],
+        mockSignatureHost: false,
+        allowUnresolvedImports: false,
+      },
+      taskHandler(parent)
```

Comment:
this is not correct. this need to be head

Reply:
this can be `context.chain.head.call('AccountNonceApi_account_nonce', accountId)`